src/components/UserSelector.tsx

```diff
@@ -73,10 +62,10 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
               onClick={() => {
-                onChange(user);
+                dispatch(authorActions.setAuthor(user));
```

Comment:
Ensure that the `setAuthor` action correctly updates the state. If the state update is asynchronous, consider handling any potential race conditions or state inconsistencies.